tests/unit/amo/pages/TestCategoriesPage.js

```diff
@@ -53,4 +74,52 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
+
+  it('returns a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
```

Comment:
should you mention that this is only for themes in the test description?

Reply:
I'd assume that `renderThemeCategories()` would be enough. The opposite test case explicitly says `does not return a 404 when addonType is not "theme"`